### PR TITLE
Issue 112 phone

### PR DESCRIPTION
**Pull request by ledenev-viktor**
add intl-phone
add phone to admin
edit validation
edit global styles

You need to decide which validation trigger to use - onChange or onBlur.
Currently in use onChange.

### CODE REVIEW

File: src/components/ui/hook-form/phone/index.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    useEffect(() => {
+        switch (phoneMeta?.iso) {
+            case 'ge':
+                setMask('.. ... .. ..');
+                break;
+            case 'ru':
+                setMask('... ... .. ..');
+                break;
+            case 'us':
```
Comment (YuliaKletsova) [inline src/components/ui/hook-form/phone/index.tsx:79]: let's remove us numbers. we do not expect people to have it

File: src/components/ui/hook-form/phone/index.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                <PhoneInput
+                    {...field}
+                    {...props}
+                    defaultMask={mask}
+                    forceDialCode={true}
+                    defaultCountry="ge"
+                    countries={countries}
+                    className={cn('inputPhone', { error: fieldState.error })}
+                    onChange={(e, data) => {
+                        console.log('data', data);
```
Comment (YuliaKletsova) [inline src/components/ui/hook-form/phone/index.tsx:100]: remove this line

File: src/styles/global-styles.ts [hunk 3]
```diff
@@ -10,7 +10,8 @@ export const globalStyles = css`
         margin: 0;
         padding: 0;
         background-color: var(--background);
-        overflow: hidden;
+        overflow-x: hidden;
+        overflow-y: visible;
```
Comment (YuliaKletsova) [inline src/styles/global-styles.ts:14]: I still don't understand why this change is needed

**Review by YuliaKletsova — CHANGES_REQUESTED** [review]

File: src/styles/global-styles.ts (hunk 3 above)
Comment (ledenev-viktor) [reply src/styles/global-styles.ts:14]: This is overflow protection. If our content goes beyond 100vh, then vertical scrolling will be added.

File: src/components/ui/hook-form/phone/index.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        switch (phoneMeta?.iso) {
+            case 'ge':
+                setMask('.. ... .. ..');
+                break;
+            case 'ru':
+                setMask('... ... .. ..');
+                break;
+            default:
+                setMask('');
+        }
```
Comment (YuliaKletsova) [inline src/components/ui/hook-form/phone/index.tsx:81]: This would be better, because we don't want to have empty mask

```suggestion
        switch (phoneMeta?.iso) {
            case 'ru':
                setMask('... ... .. ..');
                break;
            case 'ge':
            default:
                setMask('.. ... .. ..');
        }
```

**Review by YuliaKletsova — APPROVED** [review]
